libs/langchain/langchain/vectorstores/dashvector.py

```diff
@@ -29,15 +29,15 @@ class DashVector(VectorStore):
     Example:
         .. code-block:: python
 
-            from langchain.vectorstores import dashvector
+            from langchain.vectorstores import DashVector
             from langchain.embeddings.openai import OpenAIEmbeddings
             import dashvector
 
-            client = dashvector.Client.init(api_key="***")
-            client.create("langchain")
+            client = dashvector.Client(api_key="***")
```

Comment:
`dashvector.Client.init(...)` is invalid code as of `dashvector==1.0.6`.
`dashvector.Client(...)` works.